OCP-4.X/cluster.yml

```diff
@@ -0,0 +1,26 @@
+---
+- hosts: orchestration
+  user: "{{ ORCHESTRATION_USER | default('root') }}"
+  roles:
+    - { role: ansible-config, when: OPENSHIFT_INSTALL | default(True) | bool }
```

Comment:
I am of the opinion that the ansible config should just be shipped in the directory where the playbook is intended to be executed from. In this case we include [one](https://github.com/redhat-performance/scale-ci-ansible/blob/master/ansible.cfg) in `scale-ci-ansible/`  This allows multiple ansible.cfg configurations for more flexibility, however comes at the cost of potentially executing a playbook outside it's intended directory without its intended ansible.cfg.